near-sdk/compilation_tests/schema_derive_invalids.stderr

```diff
@@ -87,7 +87,7 @@ error[E0277]: the trait bound `Inner: JsonSchema` is not satisfied
             i128
           and $N others
 note: required by a bound in `SchemaGenerator::subschema_for`
- --> $CARGO/schemars-0.8.17/src/gen.rs
+ --> $CARGO/schemars-0.8.19/src/gen.rs
```

Comment:
If you rerun, you get 20. Please consider either updating to 20 or making the test independent of the version / fixing schemars version